src/components/charts/common-chart-options.js

```diff
@@ -1,6 +1,6 @@
 /* this contains global options for all charts */
-class ChartOptions {
-    constructor(theme, title, type) {
+class CommonChartOptions {
```

Comment:
I've renamed the class and the file to reflect the fact that this is now common chart options only, and to differentiate it from the `SpecificChartOptions` class.